as-modules/embedded/pom.xml

```diff
@@ -57,6 +57,11 @@
          <groupId>org.infinispan</groupId>
          <artifactId>infinispan-jcache</artifactId>
       </dependency>
+      
+      <dependency>
```

Comment:
I think this can be removed. infinispan-jcache-commons is a transitive dependency of infinispan-jcache.